WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -28,12 +28,12 @@ def auth_results
   def scramble_program
     render json: {
       "current" => {
-        "name" => "TNoodle-WCA-0.14.0",
+        "name" => "TNoodle-WCA-0.15.0",
```

Comment:
This could get interesting... do you know if the new "name" after @suushiemaniac's changes will match this?

Reply:
I think so. The is from the 0.15.0 `version.json` (obviously without the code from this PR). We have a running_version named properly.

```
{
  "current": {
    "name": "TNoodle-WCA-0.14.0",
    "information": "https://www.worldcubeassociation.org/regulations/scrambles/",
    "download": "https://www.worldcubeassociation.org/regulations/scrambles/tnoodle/TNoodle-WCA-0.14.0.jar"
  },
  "running_version": "TNoodle-WCA-0.15.0",
  "allowed": [
    "TNoodle-WCA-0.14.0"
  ],
  "history": [
    "TNoodle-0.7.4",
   .
   .
   .
```